blueocean-config/src/main/resources/features.properties

```diff
@@ -0,0 +1,10 @@
+# This file defines the enabled features
+# Any of these features can be overriden on the command line via environment variable or java system property.
+#
+# e.g. if you have a feature such as "my.cool.feature=value" you can pass "-Dfeature.my.cool.feature=anothervalue"
+# or set the environment variable "FEATURE_MY_COOL_FEATURE=ANOTHERVALUE"
+#
+# Features are looked up first by environment variable and then by system property. If no value is found, then
+# features will be resolved to the value as specified in this file
+
+pipeline.editor.enabled=false
```

Comment:
If this is a real property maybe just name it `pipeline.editor`. Surely the `enabled` part is implicit.